extensions/amp-date-display/1.0/test/test-amp-date-display.js

```diff
@@ -83,6 +83,7 @@ describes.realWin(
     });
 
     afterEach(() => {
+      clock.runAll();
```

Comment:
Makes sense, perhaps we should add some additional enforcement to ensure this happens for all tests with a `preact` implementation?

Reply:
Yeah. I'm looking for defense in depth right now. But no good ideas yet.